app/views/galleries/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1><%= @gallery.name %></h1>
+
+<p>NPO Status:
+<% if @gallery.non_profit == true %>
```

Comment:
This will benefit the user experience to make it easier to read rather than true and false.